modules/sample-audit-log-message.adoc

```diff
@@ -58,3 +63,20 @@ The log message has the following format:
   }
 }
 ----
+
+The source IP address of the request is displayed in the source fields, which makes it easier for you to investigate audit log requests and identify their origin.
```

Comment:
```suggestion
The source IP address of the request displayed in the source fields makes it easier to investigate audit log requests and identify their origin.
```

Reply:
Up to you...

Reply:
In case you use the term _parameters_: 
```suggestion
The source IP address of the request is displayed in the source parameters, which makes it easier for you to investigate audit log requests and identify their origin.
```